AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs

```diff
@@ -411,5 +411,29 @@ private static void RetrieveFirstAndLastNameFromExternalPrincipal(ExternalLoginI
             firstName = array[0];
             lastName = array[1];
         }
+        
+        //Area Names
+        private const string ADMIN_AREA = "Admin";
+
+        //Controller Names
+        private const string HOME_CONTROLLER = "Home";

```

Comment:
@nedruk, unless the string "HomeController" is used more than once in the class, then you can just use the string... I'd argue that even if it's used more than once, still use "HomeController". We should be avoiding constants unless we absolutely need them.

@stevejgordon, I didn't realize that MS is guiding towards not using nameof() for action names on controller's. Is this b/c of attributes like `[Route("STRING")]`? I'm only asking b/c we're using nameof() all over our code base to resolve action names on controllers instead of using a string with the name of the action.